### PR TITLE
[FIX] Wrong JSON key for lastModified

### LINKED ISSUE #64: [BUG] Wrong JSON key used for lastModified in Subscriptions.kt
The `lastModified` field is mapped to the `lm` key in the `Subscriptions.kt` file [here](https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK/blob/99b4ce6c2e79a7c6f7a4d2a9e3a8b2b719b94720/core/src/main/kotlin/chat/rocket/core/internal/model/Subscription.kt#L22). Both the documentation and the actual response received use the `ls` key.

I can send a PR, but since it's a tiny change it might be better to simply include it in a soon-to-be-merged one.

### PR DESCRIPTION
**Pull request by TheGamer007**
Fixes #64

### CODE REVIEW

**Comment by rafaelks** [comment]
@luciofm Can you confirm that our REST is using `ls` instead of `lm`?

**Comment by rafaelks** [comment]
@TheGamer007 Thanks for the PR! 👍

File: core/src/main/kotlin/chat/rocket/core/internal/model/Subscription.kt [hunk 1]
```diff
@@ -19,7 +19,7 @@ data class Subscription(
         @Json(name = "fname") override val fullName: String?,
         @Json(name = "ro") override val readonly: Boolean? = false,
         @Json(name = "ts") @ISO8601Date val timestamp: Long?,
-        @Json(name = "lm") @ISO8601Date val lastModified: Long?,
+        @Json(name = "ls") @ISO8601Date val lastModified: Long?,
```
Comment (luciofm) [inline core/src/main/kotlin/chat/rocket/core/internal/model/Subscription.kt]: please rename the variable to `lastSeen` also.

Comment (TheGamer007) [reply core/src/main/kotlin/chat/rocket/core/internal/model/Subscription.kt]: @luciofm Done

**Comment by luciofm** [comment]
thanks @TheGamer007

**Pull request merged**